gui/shared/lists.gui

```diff
@@ -0,0 +1,2103 @@
+
+										hbox = {
+
+											icon = {
+												name = "hook"
+												visible = "[Character.PlayerHasHooksOrHookableSecrets]"
+												size = { 25 25 }
+												framesize = { 40 40 }
+												texture = "gfx/interface/icons/portraits/hook_secret.dds"
+
+												frame = "[Character.GetHookOrHookableSecretsFrame]"
+												tooltip = "[Character.HookTooltip]"
+											}
+
+											icon = {
+												name = "icon_non_dejure"
+												visible = "[Not( VassalContract.IsRightfulLiege )]"
+												size = { 30 30 }
+												texture = "gfx/interface/icons/levy_tax_reduction.dds"
+
+												using = tooltip_ws
+												tooltip = "MRW_TT_NOT_RIGHTFUL_LIEGE"
+											}
+										}
+										expand = {}
+									}
+								}
+								expand = {}
+							}
+						}
+					}
+
+					expand = {}
+
+					hbox = {
+						layoutpolicy_horizontal = expanding
+						spacing = 5
+
+						text_single = {
+							text = "REALM_WINDOW_SIZE"
+							tooltip = "REALM_WINDOW_SIZE_TT"
+							align = nobaseline
+							visible = "[NotEqualTo_int32(Character.GetPrimaryTitle.GetTierFrame, '(int32)2')]" #nonbaron
+
+						}
+
+						hbox = {
+							spacing = 3
+							margin = { 5 0 }
+
+							block "own_vassal"
+							{
+								visible = no
+							}
+
+							icon = {
+								size = { 22 22 }
+								texture = "gfx/interface/icons/icon_soldier.dds"
+							}
+
+							text_single = {
+								layoutpolicy_horizontal = expanding
+								text = "[Character.GetMilitaryStrengthText]"
+								align = nobaseline
+								default_format = "#high"
+							}
+						}
+
+						text_single = {
+							visible = "[Character.IsPowerfulVassal]"
+							text = "MY_REALM_WINDOW_POWERFUL_VASSAL"
+							layoutpolicy_horizontal = expanding
+							align = nobaseline
+						}
+
+						expand = {}
+
+						hbox_skills_grid = {}
+					}
+
+					expand = {}
+				}
+			}
+
+			block "own_vassal"
+			{
+				block "feudal_contract"
+				{
+					vbox = {
+						name = "feudal_info"
+						layoutpolicy_horizontal = expanding
+						margin = { 10 5 }
+						spacing = 10
+
+						background = {
+							using = Background_Area
+						}
+
+						hbox = {
+							layoutpolicy_horizontal = expanding
+							visible = "[And( VassalContract.UsesObligationLevels, Not( Character.GetPrimaryTitle.IsBarony))]"
+							spacing = 10
+
+							button_round = {
+								enabled = "[Character.IsPlayerInteractionValid('liege_modify_vassal_contract_interaction')]"
+								onclick = "[Character.OpenPlayerInteraction('liege_modify_vassal_contract_interaction')]"
+								tooltip = "[Character.GetPlayerInteractionTooltip('liege_modify_vassal_contract_interaction')]"
+
+								button_feudal_contract = {
+									alwaystransparent = yes
+									parentanchor = center
+								}
+							}
+
+							text_single = { # Display the usual Levy and Tax for Feudal
+								#visible = "[Character.GetGovernment.IsType('feudal_government')]"
```

Comment:
This line is commented out, but the vanilla 1.6.0 file doesn't comment it. Why are we messing with this?

Reply:
idk, i used script for showing prowess